### PR TITLE
Support top N holders 25

### PR DESCRIPTION
**Pull request by lzxm160**
work on #25 
add:
query{
 topHolders(endEpochNumber:3,numberOfHolders:10){
  address,balance
}
}

### CODE REVIEW

File: queryprotocol/actions/protocol.go [hunk 1]
```diff
@@ -366,6 +372,47 @@ func (p *Protocol) GetXrc20ByPage(numPerPage, page uint64) (cons []*Xrc20Info, e
...
+	if err != nil {
+		return nil, errors.Wrap(err, "failed to parse results")
+	}
+	if len(parsedRows) == 0 {
+		err = indexprotocol.ErrNotExist
+		return nil, err
+	}
+	for _, parsedRow := range parsedRows {
+		r := parsedRow.(*TopHolders)
+		holder := &TopHolders{
```
Comment (lizhefeng) [inline queryprotocol/actions/protocol.go]: Is this redundant?

Comment (lzxm160) [reply queryprotocol/actions/protocol.go]: There's import cycles if I don't redefine this

Comment (lizhefeng) [reply queryprotocol/actions/protocol.go]: I mean, can we do this:
```
holders = append(holders, parsedRow.(*TopHolders))
```

Comment (lzxm160) [reply queryprotocol/actions/protocol.go]: > I mean, can we do this:
> 
> ```
> holders = append(holders, parsedRow.(*TopHolders))
> ```

I'll try it

File: graphql/schema.graphql [hunk 2]
```diff
@@ -6,6 +6,12 @@ type Query {
     hermes(startEpoch: Int!, epochCount: Int!, rewardAddress: String!, waiverThreshold: Int!): Hermes
     xrc20: Xrc20
     action: Action
+    topHolders(endEpochNumber: Int!, numberOfHolders: Int!):[TopHolders]!
+}
+
+type TopHolders{
```
Comment (lizhefeng) [inline graphql/schema.graphql]: TopHolder

**Pull request merged**